README.md

```diff
@@ -74,3 +74,21 @@ To get complete documentation for each command and its parameters use the `--hel
   - This is an asynchronous script that distibutes all unclaimed rewards.
   - If any of the transactions failed to complete, or returned an error, simply run the script again to retry all incomplete distributions.
   - Rewards are distributed fully when the output json `distribution-<distributor-address>.json` and `rewardsByAddress.json` mapping accounts keys match (same number of lines in each json file).
+
+
+  ## Use CLI to verify merkle root of a deployed Merkle Distribution contract for a rewards round
+  #### 1) Follow steps 1 and 2 from above so you're ready to compute a merkle root
+  #### 2) Follow step 3 and use all the parameters given for the rewards round plus the verifyAgainstContract parameter with the contract address used in the rewards round. Parameters can be found in recent discussion on the rewards forum post [found here](https://forum.celo.org/t/governance-proposal-to-reward-early-users/662/41).
+  ```bash
+     ./bin/run rewards:generatemerkle \
```

Comment:
nit: all of our other commands use kebab-case "generate-merkle"

Reply:
oh I see. (lol kebab case) The first word is always how I did it, and I went off that, not realizing after the colon it's kebab. Maybe I can make this a wider fix later.